client/ayon_flame/plugins/publish/collect_audio.py

```diff
@@ -0,0 +1,34 @@
+import pyblish
+
+
+class CollectAudio(pyblish.api.InstancePlugin):
+    """Collect new audio."""
+
+    order = pyblish.api.CollectorOrder - 0.48
+    label = "Collect Audio"
+    hosts = ["flame"]
+    families = ["audio"]
+
+    def process(self, instance):
+        """
+        Args:
+            instance (pyblish.Instance): The shot instance to update.
+        """
+        # Retrieve instance data from parent instance shot instance.
+        parent_instance_id = instance.data["parent_instance_id"]
+        edit_shared_data = instance.context.data["editorialSharedData"]
+        instance.data.update(
+            edit_shared_data[parent_instance_id]
+        )
+
+        if instance.data.get("reviewTrack") is not None:
+            instance.data["reviewAudio"] = True
+            instance.data.pop("reviewTrack")
```

Comment:
This might be weird question, but why if `reviewTrack` popped out from data in `CollectAudio`?

Reply:
I'm not entirely sure to be honest. this is something I've reported from Hiero implementation for consistency.
Initial logic comes from here: https://github.com/ynput/ayon-hiero/blob/0.2.3/client/ayon_hiero/plugins/publish/precollect_instances.py#L303

It makes sense to me that a review track is not needed when dealing with audio, so I assumed this was to prevent side-effects further down..

Reply:
And is there any plugin that does set `"audio"` family?

Reply:
> And is there any plugin that does set `"audio"` family?

This is quite different audio collector. This one is collecting audio otio clips for transcoding audio product's representation.

Tho I am not sure this will currently do anything at all. It seems to me that we are not including audio clips from audio tracks inside of `flame_export.py` **OTIO**.  Perhaps this part of the otio exporter is blocking audio segments to be included?